### PR TITLE
Update various component geometry types

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues

- https://github.com/cityofaustin/atd-data-tech/issues/14408

After this change, I will manually update project components in Moped that need to be re-mapped against their new geometry type. This query will reveal that todo list:

```sql
SELECT
    *
FROM
    moped_proj_components mpc
    LEFT JOIN moped_components mc ON mc.component_id = mpc.component_id
WHERE
    mpc.component_id in (25, 27, 40, 48, 49, 50, 57, 58, 2, 21, 35, 45);
```

## Testing

**URL to test:**  Local

1. Create any of the component types listed in the issue. They should follow the following geometry type:
 
- Dynamic Speed Display Device - line
- Highway - Access Ramp - line
- Pavement Marking - School Zone - line
- Speed Management - Speed Cushions (Asphalt) - line
- Speed Management - Speed Cushions (Rubber) - line
- Speed Management - Speed Humps - line
- Pavement Marking - Sharrows - line
- Pavement Marking - Sharrows (Wide Curb Lane) - line
- Access Control - Driveway Modification - point
- Bike Lane - Turn Lane - point
- Placemaking - point
- Sidewalk - Ramps - point

2. Observe that we have a new **Signage - linear** component type.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by johnclary** [comment]
i'm reviewing the review request on this for now.

File: moped-editor/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
   "license": "CC0-1.0",
-  "version": "1.36.0",
+  "version": "1.35.1",
```
Comment (johnclary) [inline moped-editor/package.json:5]: we're going to ship this as a patch!

File: moped-database/migrations/1698428471744_create_signage_line_component/up.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
+-- reset coponent id sequence to max component ID
+SELECT
+    setval('moped_components_component_id_seq', (
+            SELECT
+                max(component_id)
+                FROM moped_components));
```
Comment (johnclary) [inline moped-database/migrations/1698428471744_create_signage_line_component/up.sql:6]: while testing locally with prod data i discovered that our component ID sequence was at 128 although we only had 97 components in the DB. my theory is that i screwed up the sequence when we had that failed/aborted migration a few releases ago.

This was also a good reminder to add the ID sequence resetter to down migrations.

File: moped-database/migrations/1698428471744_create_signage_line_component/down.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+-- delete component (will cascade to component_work_types)
+DELETE from public.moped_components WHERE
+        moped_components.component_name = 'Signage'
+        AND moped_components.component_subtype = 'linear';
+
+-- reset coponent id sequence
```
Comment (chiaberry) [inline moped-database/migrations/1698428471744_create_signage_line_component/down.sql:6]: *component

File: moped-database/migrations/1698428471744_create_signage_line_component/up.sql [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+-- create new component
+INSERT INTO public.moped_components (
+        component_name,
+        component_subtype,
+        line_representation,
+        feature_layer_id
+    ) VALUES
+    ('Signage', 'linear', TRUE, 4 );
```
Comment (chiaberry) [inline moped-database/migrations/1698428471744_create_signage_line_component/up.sql:15]: our other subtypes have the first letter capitalized

**Review by mddilley — COMMENTED** [review]
Gave this a quick eyeball check so I could approve the patch, and this looks great to me!

**Pull request merged**